cmd/tk/util.go

```diff
@@ -15,31 +15,36 @@ func pageln(i ...interface{}) {
 	fPageln(strings.NewReader(fmt.Sprint(i...)))
 }
 
-// fPageln invokes the systems pager with the supplied data
-// falls back to fmt.Println() when paging fails or non-interactive
+// fPageln invokes the systems pager with the supplied data.
+// If the PAGER environment variable is empty, no pager is used.
+// If the PAGER environment variable is unset, use less with posix flags.
```

Comment:
What is POSIX less here? Any reference at hand?

Reply:
I was actually getting muddled, `more` has POSIX spec (https://pubs.opengroup.org/onlinepubs/9699919799/) and `less` can be run in a `more`-ish mode. Humorously, this can be triggered with the environment variable `LESS_IS_MORE=1`. Perhaps it would be more truthful to say:
> If the PAGER environment variable is unset, use GNU `less` with convenience flags

Reply:
Sounds right!